src/examples/pane/IconButtonPosition.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { PaneProvider, Pane } from '@zendeskgarden/react-grid';
+import useResizeObserver from 'use-resize-observer';
+import { XL, Paragraph } from '@zendeskgarden/react-typography';
+import styled from 'styled-components';
+
+const StyledDiv = styled.div`
+  padding: ${p => p.theme.space.base * 6}px;
+`;
+
+const StyledParagraph = styled(Paragraph)`
+  margin-top: ${p => p.theme.space.xs};
+`;
+
+const StyledPaneContent = styled(Pane.Content)`
+  height: 180px;
+`;
+
+const StyledPanesContainer = styled.div`
+  & > :first-child {
+    background-color: ${p => p.theme.palette.grey[100]};
```

Comment:
[nit] prefer the `getColor` utility (and `'neutralHue'`) to show pristine Garden system usage.

Reply:
Updated in https://github.com/zendeskgarden/website/pull/416/commits/f00aab8b4acb4eb8cd3dc995da1153e5fc5c10a3.